plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorPart.java

```diff
@@ -1155,7 +1155,7 @@ public void run()
         {
             getDiagram().setAttributeStyle(style, !isChecked());
             refreshEntityAndAttributes();
-            refreshDiagram(true, false);
+            refreshDiagram(true, true);
```

Comment:
That's probably not part of the current issue, but we lose user's layout on refreshing the diagram (Keep layout checked)